Gems/StartingPointCamera/Code/Source/CameraLookAtBehaviors/SlideAlongAxisBasedOnAngle.cpp

```diff
@@ -23,7 +23,9 @@ namespace Camera
                 ->Version(1)
```

Comment:
update: As you're changing the serialized data you'll need to bump the version number here...

In some cases it's also necessary to write a version converter (to update the old data to the new format), but if this isn't widely used and we're not yet committing to stability this change might be okay - worth double checking though with who owns this component/gem)